src/components/general/SideSheet/Modal/index.tsx

```diff
@@ -31,7 +33,11 @@ export default ({
     headerIcons,
     size = 'xlarge',
```

Comment:
Considering we have changed the sizes, should we set the default to 'large' instead of 'xlarge' so existing implementations don't change size unexpectedly?

Reply:
Good idea. Fixed